src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -843,6 +839,7 @@ public void UpdateCacheIfNeeded()
             protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, bool> activeOutput, out Action disposer)
             {
                 disposer = null;
+                _parent._session.graph.as_default();
```

Comment:
>_parent._session.graph.as_default(); [](start = 16, length = 36)

is this needed?

Reply:
Yes, test Microsoft.ML.Scenarios.TensorFlowScenariosTests.TensorFlowImageClassification is failing without this

---
In reply to: [324938993](https://github.com/dotnet/machinelearning/pull/4205#discussion_r324938993) [](ancestors = 324938993)